src/pages/ShareCodePage.js

```diff
@@ -42,7 +42,9 @@ class ShareCodePage extends React.Component {
 
     render() {
         const isReport = this.props.report != null && this.props.report.reportID != null;
-        const subtitle = ReportUtils.getChatRoomSubtitle(this.props.report);
+        const title = isReport ? ReportUtils.getReportName(this.props.report) : this.props.currentUserPersonalDetails.displayName;
+        const formattedEmail = this.props.formatPhoneNumber(this.props.session.email);
+        const subtitle = isReport ? ReportUtils.getChatRoomSubtitle(this.props.report) || ReportUtils.getParentNavigationSubtitle(this.props.report).rootReportName : formattedEmail;
```

Comment:
```suggestion
        const subtitle = isReport ? (ReportUtils.getChatRoomSubtitle(this.props.report) || ReportUtils.getParentNavigationSubtitle(this.props.report).rootReportName) : formattedEmail;
```
just to make this a little easier to understand

Reply:
weird! never mind then haha